app/mailers/school_certifying_officials_mailer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class SchoolCertifyingOfficialsMailer < TransactionalEmailMailer
+  SUBJECT = 'Applicant for VA Rogers STEM Scholarship'
+  GA_CAMPAIGN_NAME = 'school-certifying-officials-10203-submission-notification'
+  GA_DOCUMENT_PATH = '/email/form'
+  GA_LABEL = 'school-certifying-officials-10203-submission-notification'
+  TEMPLATE = 'school_certifying_officials'
+
+  STAGING_RECIPIENTS = %w[
+    Delli-Gatti_Michael@bah.com
+    roth_matthew@bah.com
+    shawkey_daniel@bah.com
+    sonntag_adam@bah.com
+  ].freeze
+
+  def build(applicant, recipients, ga_client_id)
+    @applicant = applicant
```

Comment:
The abstraction might read better with an "Applicant" model class, where you can validate that `applicant` contains the data you expect, and you can also use it to house the `first_initial_last_name` logic. e.g.

```ruby
@applicant = EdithNourseApplicant.new(applicant)
raise "Invalid Applicant: #{@applicant.errors}"

### edith_nourse_applicant.rb
def initialize(applicant)
  self.first_name = applicant[:veteranFullName][:first]
  self.last_name = applicant[:veteranFullName][:last]
end

def first_initial_last_name
  return '' if first_name.nil? || last_name.nil?

  "#{first_name.first[0, 1]} #{last_name.last}"
end


### app/mailers/views/school_certifying_officials.html.erb
Name of Student: <%=@applicant.first_initial_last_name %>
```

Reply:
i put that `first_initial_last_name` in `app/mailers/transactional_email_mailer.rb` because i was treating that class the same as `app/workers/education_form/forms/base.rb` is for spool files